protocol-designer/src/flex-components/LandingPage.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
+import { Icon, OutlineButton } from '@opentrons/components'
+import { Link } from 'react-router-dom'
+import { i18n } from '../localization'
+import { StyledText } from './protocol-editor/StyledText'
+import styles from './protocol-editor/FlexComponents.css'
+
+interface Props {
```

Comment:
Maybe rename this to `NavDataProps` so it is easier to tell what the Props are for.

```suggestion
interface NavDataProps {
```